translator/translate/otel/pipeline/appsignals/translator.go

```diff
@@ -54,6 +55,10 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	translators.Processors.Set(resourcedetection.NewTranslator(resourcedetection.WithDataType(t.dataType)))
 	translators.Processors.Set(awsappsignals.NewTranslator(awsappsignals.WithDataType(t.dataType)))
 
+	if enabled, _ := common.GetBool(conf, common.DebugLogging); enabled {
```

Comment:
Still think there's risk to enabling this level of debug logging using the existing `agent/debug` field. We can determine if we need to change this before we release it.